zio/zeekio/types.go

```diff
@@ -0,0 +1,17 @@
+package zeekio
+
+import (
+	"strings"
+)
+
+func zeekTypeToZng(typ string) string {
+	// As zng types diverge from zeek types, we'll probably want to
+	// re-do this but lets keep it simple for now.
+	typ = strings.ReplaceAll(typ, "string", "bstring")
+	return strings.ReplaceAll(typ, "vector", "array")
+}
+
+func zngTypeToZeek(typ string) string {
+	typ = strings.ReplaceAll(typ, "bstring", "string")
+	return strings.ReplaceAll(typ, "array", "vector")
+}
```

Comment:
I think this is too aggressive.  zng record type can contain arbitrary names, e.g. `record[mystring:string]`, this would rename the field.
Maybe its safe in this case since you're only using it when writing a record that has already been flattened?  But if that's a case, there should at least be a big warning comment here explaining that this function isn't general purpose.

Reply:
@aswan that's a good point.